src/sb-util.ts

```diff
@@ -42,6 +43,7 @@ export class ScratchProject {
         storage.set(this, projectJSON);
     }
 
+    // eslint-disable-next-line
```

Comment:
Can we disable a specific rule?

Reply:
yes just add rule name after :) (please do @evmiguel )